### PR TITLE
Prediction settings

### PR DESCRIPTION
**Pull request by eggrobin**
As requested by @diomedea, add a toggle to display the patched conics for comparison.

### CODE REVIEW

**Comment by diomedea** [comment]
Many thanks, egg!

File: ksp_plugin_adapter/ksp_plugin_adapter.cs [hunk 1]
```diff
@@ -341,18 +356,38 @@ where is_in_physics_bubble(vessel)
...
-          active_vessel.DetachPatchedConicsSolver();
-          active_vessel.patchedConicRenderer = null;
+        }
+        if (display_patched_conics_) {
+          foreach (PatchRendering patch_rendering in
+                   active_vessel.patchedConicRenderer.patchRenders) {
+            patch_rendering.visible = true;
+          }
+          for (int i = 1;
+               i < active_vessel.patchedConicRenderer.flightPlanRenders.Count;
```
Comment (pleroy) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Any reason why this is not a foreach?

File: ksp_plugin_adapter/ksp_plugin_adapter.cs [hunk 2]
```diff
@@ -430,7 +465,10 @@ where is_in_physics_bubble(vessel)
     rendered_trajectory_.layer = 31;
     rendered_prediction_ = new VectorLine(
         lineName     : "rendered_prediction_",
-        linePoints   : new UnityEngine.Vector3[kLinePoints],
+        linePoints   : new UnityEngine.Vector3[
+                           (int)(prediction_lengths_[prediction_length_index_] /
+                                 prediction_steps_[prediction_step_index_]) *
+                           2],
```
Comment (pleroy) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Move the 2 to the beginning.

File: ksp_plugin_adapter/ksp_plugin_adapter.cs (hunk 1 above)
Comment (eggrobin) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: kinda.

File: ksp_plugin_adapter/ksp_plugin_adapter.cs (hunk 2 above)
Comment (eggrobin) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Done.

**Pull request merged**